airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -116,6 +116,14 @@ public class KubePodProcess implements KubePod {
   private static final ResourceRequirements DEFAULT_SOCAT_RESOURCES = new ResourceRequirements()
       .withMemoryLimit(configs.getSidecarKubeMemoryLimit()).withMemoryRequest(configs.getSidecarMemoryRequest())
       .withCpuLimit(configs.getSocatSidecarKubeCpuLimit()).withCpuRequest(configs.getSocatSidecarKubeCpuRequest());
+  private static final String DD_ENV_VAR =
+      "-XX:+ExitOnOutOfMemoryError -javaagent:/airbyte/dd-java-agent.jar -Ddd.profiling.enabled=true -XX:FlightRecorderOptions=stackdepth=256 -Ddd.trace.sample.rate=30 -Ddd.trace.request_header.tags=User-Agent:http.useragent";
```

Comment:
This is duplicated between `DockerProcessFactory` and `KubeProdProcess`.  Consider extracting to a common place so we don't have to remember to update it in two places should the command change.